code/modules/reagents/Chemistry-Reagents.dm

```diff
@@ -497,7 +507,8 @@ datum
 								M.IgniteMob()			//Only problem with igniting people is currently the commonly availible fire suits make you immune to being on fire
 								M.adjustFireLoss(3)		//Hence the other damages... ain't I a bastard?
 								M.mind.vampire.nullified = max(5, M.mind.vampire.nullified + 2)
-				holder.remove_reagent(src.id, 10 * REAGENTS_METABOLISM) //high metabolism to prevent extended uncult rolls.
+				holder.remove_reagent(src.id, 0.4)	//fixed consumption to prevent balancing going out of whack

```

Comment:
Couldn't you just set the metabolization_rate to 0.4 and avoid needing this here?